### PR TITLE
Nilesh

### PR DESCRIPTION
**Pull request by Nilesh2710**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import numpy as np
+import pandas as pd
+
+from sklearn.model_selection import train_test_split
```
Comment (chetan-bangale1) [inline utils.py:4]: Why do we need to split the data ?

Comment (chetan-bangale1) [inline utils.py:4]: Is it helping us ?

Comment (Nilesh2710) [reply utils.py:4]: For training & testing

Comment (Nilesh2710) [reply utils.py:4]: Yes

Comment (chetan-bangale1) [reply utils.py:4]: ok

Comment (chetan-bangale1) [reply utils.py:4]: ok

**Pull request merged**